### DISCUSSION TITLE
How can I edit the search results in the start menu?

### DISCUSSION THREAD

**Discussion by adabo** [Open-Shell/Open-Shell-Menu#873] [Q&A] [answered]
I've noticed that Open-Shell remembers the link I opened after I typed in some keyword. Then it remembers that and suggests it on the top in subsequent results. Well, sometimes I accidentally open the wrong result and now it is not the first result and if I press enter on my keyboard it will open that first result because the highlighted text is always on top. So can you see my predicament? I want to edit whatever database is storing this information so that I can correct my error and put the right result on the top so that when I press enter on the keyboard it will open the top link that I meant to open the first time I clicked it.

Does any of this make sense? Thanks for your help.

**Comment by Gaurav-Original-ClassicShellTester** [comment, marked as answer]
Exit the Start menu, then delete the ItemRanks DWORD from HKEY_CURRENT_USER\Software\OpenShell\StartMenu and start the menu again or re-logon.

Or you can just hold down the Shift key while launching something to keep the Start menu open. Launch the correct item more number of times and it will rise to the top.

To entire turn off the feature, in Settings -> Search box tab -> uncheck "Track frequency of use".

**Reply by adabo** [reply]
Works, thanks (registry method)! I like the feature, actually. Means I don't have to type as many letters to launch a program, thanks to its rank.
